src/storage/config.rs

```diff
@@ -18,6 +18,7 @@ const DEFAULT_GC_RATIO_THRESHOLD: f64 = 1.1;
 const DEFAULT_MAX_KEY_SIZE: usize = 4 * 1024;
 const DEFAULT_SCHED_CAPACITY: usize = 10240;
 const DEFAULT_SCHED_CONCURRENCY: usize = 2048000;
+const MAX_SCHED_CONCURRENCY: usize = 40960000;
```

Comment:
what is the time when we create the vector with this size?

Reply:
About 6 seconds.

Reply:
Could you leave some comments in this file? Otherwise, it looks like a magic value.